### ISSUE TITLE
[mac] feature request: automatic update

### ISSUE THREAD

**Issue by MayuraVerma** [keymanapp/keyman#952]
Please add automatic update feature to mac version of keyman.

Amongst all the platform, mac version is the only that doesn't have this feature.

Please add it.

**Comment by mcdurdin** [comment]
Yes, agreed. We'd like to make it possible to deploy via app store, but app store doesn't support the APIs we need for keyboard integration. I don't have a target version for this feature yet.

**Comment by mcdurdin** [comment]
* For app, https://sparkle-project.org/
* For keyboards, internal UI required.
